osu.Game/Graphics/UserInterface/BackButton.cs

```diff
@@ -137,13 +145,17 @@ protected override bool OnClick(InputState state)
         {
             var flash = new Box
             {
-                RelativeSizeAxes = RelativeSizeAxes,
-                Colour = colorBright,
+                RelativeSizeAxes = Axes.None,

```

Comment:
why not just use relative here? also, if you're using the same variable in more than one places you *have* to make it a constant (or member variable)